### PR TITLE
Migrate DTL infrastructure and deployment changes

### LINKED ISSUE #554: Migrate DTL infrastructure and deployment changes
#### Problem description

As part of the process of standing up the DTL instance of DistrictBuilder, we made a handful of infrastructure and deployment changes. Ensure that those are ported to this repository.

- `variables.tf`
- CloudWatch Logs configuration via `cloud-config`
- Private DNS records for application server
- Remove CloudFormation JSON
- Add CI override file for Docker Compose
- Use centrally managed Geoserver container image (`FROM quay.io/azavea/geoserver:2.11.2-tomcat9-jre8-slim)`
- Various changes in `scripts`
- ...

#### Replication Steps

- Diff the two repositories
- Note all of the differences

#### Expected Behavior or Output

- All of the desirable changes are reconciled

### PR DESCRIPTION
**Pull request by rbreslow**
## Overview

As part of the process of standing up the DTL instance of DistrictBuilder, we made a handful of infrastructure and deployment changes. Ensure that those are ported to this repository.

- `variables.tf`
- CloudWatch Logs configuration via `cloud-config`
- Private DNS records for application server
- Remove CloudFormation JSON
- Add CI override file for Docker Compose
- Use centrally managed Geoserver container image (`FROM quay.io/azavea/geoserver:2.11.2-tomcat9-jre8-slim)`
- Various changes in `scripts`
- ...

The approach taken here was to create a branch off of `develop` on the head fork [azavea/district-builder-dtl-pa](https://github.com/azavea/district-builder-dtl-pa) and drop all commits that weren't related to infrastructure changes via an interactive rebase. Then, the branch was merged with the base fork `develop` branch and conflicts were resolved.

Closes #554

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # DistrictBuilder
 
-[![Build Status](https://travis-ci.org/PublicMapping/DistrictBuilder.svg?branch=develop)](https://travis-ci.org/PublicMapping/DistrictBuilder)
+[![Build Status](https://travis-ci.org/PublicMapping/DistrictBuilder.svg?branch=develop)](https://travis-ci.org/azavea/district-builder-dtl-pa)
```
Comment (hectcastro) [inline README.markdown]: I think that we want to keep the previous state here.

File: deployment/terraform/variables.tf [hunk 2]
```diff
@@ -21,3 +21,18 @@ variable "ssl_certificate_arn" {}
...
+variable "is_subdomain" {}
+
+variable "ssh_identity_file_path" {
+  default = "~/.ssh/district-builder.pem"
+}
+
+variable "cloudfront_aliases" {
+  description = "List of CNAMES that will refer to this project"
+  default     = []
+}
```
Comment (hectcastro) [inline deployment/terraform/variables.tf:38]: I don't think that CI will work without the encrypted PEM file that was deleted below. We need this to be able to establish SSH connections from Travis.

File: django/publicmapping/publicmapping/settings.py [hunk 3]
```diff
@@ -75,6 +79,29 @@
...
+    # Rollbar setup
+    ROLLBAR = {
+        'access_token': ROLLBAR_SERVER_TOKEN,
+        'environment': ENVIRONMENT,
+        'branch': 'master',
+        'root': '/usr/src/app',
+    }
+    MIDDLEWARE.append('rollbar.contrib.django.middleware.RollbarNotifierMiddleware')
+
+SERIALIZATION_MODULES = {
```
Comment (hectcastro) [inline django/publicmapping/publicmapping/settings.py]: Based on the blame in the other repository, this change was added as part of a management command that doesn't exist here. Based on that, it doesn't seem needed. May need to get feedback from @pcaisse to confirm.

See: https://github.com/azavea/district-builder-dtl-pa/commit/ad11eadd6059adb1f5b20f805c1433aac0dc846e

**Review by hectcastro — COMMENTED** [review]
Good approach to keeping some of the original commits together.

File: README.markdown (hunk 1 above)
Comment (rbreslow) [reply README.markdown]: This change should be the other way around. I think there's something wrong with the version history.

Comment (hectcastro) [reply README.markdown]: If I pull down the branch locally, it contains the DTL repository name:

```
$ head README.markdown
# DistrictBuilder

[![Build Status](https://travis-ci.org/PublicMapping/DistrictBuilder.svg?branch=develop)](https://travis-ci.org/azavea/district-builder-dtl-pa)

[DistrictBuilder](http://www.districtbuilder.org/) is software created by the [Public Mapping Project](http://www.publicmapping.org/resources/software).

The development environment is docker-compose for services inside a Vagrant virtual machine.

## Table of Contents
  * [Installation](#installation)
```

File: django/publicmapping/publicmapping/settings.py (hunk 3 above)
Comment (rbreslow) [reply django/publicmapping/publicmapping/settings.py]: After looking at the history, and inspecting the contents of https://github.com/PublicMapping/DistrictBuilder/pull/553, I don't think there's anything that would depend on this.

**Review by hectcastro — COMMENTED** [review]
👍

File: docker-compose.yml [hunk 4]
```diff
@@ -87,4 +86,3 @@ volumes:
   data:
   sld:
   tmp:
-  translations:
```
Comment (hectcastro) [inline docker-compose.yml:90]: There is still a reference to this volume, which is causing CI to fail.

**Pull request merged**